framework/src/modules/chain/helpers/config.js

```diff
@@ -104,7 +104,12 @@ function Config(packageJson, parseCommandLineOptions = true) {
 
 	let customConfig = {};
 	if (program.config || process.env.LISK_CONFIG_FILE) {
-		customConfig = loadJSONFile(program.config || process.env.LISK_CONFIG_FILE);
+		const customConfigPath = path.join(
+			__dirname,
+			'../../../../../',
```

Comment:
This looks more like a workaround more than a fix of the actual issue.

Reply:
Patch it for now.
Other paths in folder are constructed in the same way.